### DISCUSSION TITLE
Vterm Serial polarity

### DISCUSSION THREAD

**Discussion by Kabak85** [lcgamboa/picsimlab#111] [Ideas]
Hello,

When you use radio transceiver for example, the idle signal must be zero (no emission) so you have to invert the polarity. 

Is it possible to add a parameter to "Vterm" in order to invert the polarity of a serial signal ?

**Comment by lcgamboa** [comment]
Not currently. Would a logic not gate at the Vterm input be enough to solve the problem? I have a plan to add logical blocks in PICSimLab.

**Comment by Kabak85** [comment]
The polarity inversion work like a "not" operator so it will also work.

**Reply by lcgamboa** [reply]
I have [updated the PICSimLab ](https://github.com/lcgamboa/picsimlab/releases)and added the Logic Blocks (Spare parts Window-> menu Others-> Logic Block). Using a NOT logic function you will get the serial tx pin polarity inverted.

**Comment by Kabak85** [comment]
Hi @lcgamboa 

I just tried the "NOT" function and it's work fine.

Thanks.
